File: Sortings/bubbleSort.py
def bubbleSort(arr):
    n =  len(arr)
    for i in range(0, n - 1):
        for j in range(0, n - i - 1):
            if arr[j] > arr[j + 1]:
                # swapping the elements
                temp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = temp
                
    return arr

# Example usage
if __name__ == "__main__":
    arr = [64, 34, 25, 12, 22, 11, 90]
    print("Unsorted array:", arr)
    sorted_arr = bubbleSort(arr)
    print("Sorted array:", sorted_arr)

File: Sortings/bubbleSort.java
package Sortings;

public class bubbleSort{
    
    public static void BubbleSort(int[] arr){
        int n = arr.length;
        for(int i = 0; i< n - 1; i ++){
            for(int j = 0; j < n - i - 1; j++){
                if(arr[j] > arr[j + 1]){
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
        System.out.println("After bubble sort:");
        for(int i = 0; i < n; i++){
            System.out.print(arr[i] + " ");
        }
    }

    public static void main(String[] args) {
        int[] arr = {13, 46, 24, 52, 20, 9};
        int n = arr.length;
        System.out.println("Before bubble sort:");
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }
        System.out.println();
        BubbleSort(arr);
    }
    
}
